apcd-cms/src/apps/view_users/templates/view_users.html

```diff
@@ -54,21 +53,14 @@ <h1>View Users</h1>
 <script>
 /*filters by organization with dropdown*/
 function filterTableByOrganization() {
-  var dropdown, filter, newinput, bestinput, table, tr, td, i;
+  var filter, xhr;
   input = document.getElementById("organizationFilter");
-  filter = input.value.replace("&", "&amp;").replace("(", "").replace(")","");
-  table = document.getElementById("usersTableBody");
-  tr = table.getElementsByTagName("tr");
-  for (i = 0; i < tr.length; i++) {
-    td = tr[i].getElementsByTagName("td")[2];
-    if (td) {
-      if (td.innerHTML.replace("(", "").replace(")","").search(filter) > -1) {
-        tr[i].style.display = "";
-      } else {
-        tr[i].style.display = "none";  
-      }
-    }  
-  }
+  filter = input.value.replace("&", encodeURIComponent('&'));
+  xhr = new XMLHttpRequest();
```

Comment:
It may make sense to eventually switch this over to an AJAX call so the view can get the filter information without reloading the page.
I did something similar for the exception threshold here . This may require follow on work to update functions in your view to get the filter param and your URL patterns. So this is just a suggestion for down the line.

Reply:
Good point! I'll make a Jira task for this.